build/ci/golangci.yml

```diff
@@ -63,7 +63,7 @@ linters:
     - errorlint
     - exportloopref
     - forbidigo
-    - gci
+    #- gci
```

Comment:
Tohle by údajně fungovat mělo i v Go 1.19, ale blbě to detekuje komentáře. 
![CleanShot 2022-10-04 at 09 12 52](https://user-images.githubusercontent.com/215660/193757778-da291c44-113e-40fd-a48b-2bb181322bf4.jpg)

Reply:
Nedari sa mi to nasimulovat, mozes to prosim skusist s vymazanym `cache` volume?
To formatovanie importov je myslim celkom dolezite, aby si to kazde IDE nerobilo podla seba.

Reply:
Vypadá to ok!